code/implementation/docker/compute/src/main/java/io/cattle/platform/docker/process/instance/DockerInstancePreCreate.java

```diff
@@ -77,6 +82,15 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
         return new HandlerResult(data).withShouldContinue(true);
     }
 
+    private Boolean removeLabel(Boolean isNativeContainer, String data_mode) {
```

Comment:
In Java, camelcase is the standard for the parameter name. So it should be dataMode